### ISSUE TITLE
Plugins broken with 1.10.14 & Python 3.8

### ISSUE THREAD

**Issue by Alien2150** [apache/airflow#13019]
**Apache Airflow version**: 1.14

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`): 1.17

**Environment**:

- **Cloud provider or hardware configuration**: Kubernetes 
- **Python Version**: 3.8
- **OS** (e.g. from /etc/os-release):
- **Kernel** (e.g. `uname -a`):
- **Install tools**:
- **Others**:

**What happened**:

Plugins will be broken when importlib-metadata is not installed with python3.8 base image.

**What you expected to happen**:

This is the stack-trace:
```
[2020-12-11 14:12:29,757] {plugins_manager.py:159} ERROR - Failed to import plugin AirflowPrometheus                                                                            │
│ Traceback (most recent call last):                                                                                                                                              │
│   File "/home/airflow/.local/lib/python3.8/site-packages/airflow/plugins_manager.py", line 150, in load_entrypoint_plugins                                                      │
│     plugin_obj.__usable_import_name = entry_point.module                                                                                                                        │
│ AttributeError: 'EntryPoint' object has no attribute 'module'
```

**How to reproduce it**:
Put `airflow-exporter==1.3.2` into requirements.txt and start the scheduler.

**Anything else we need to know**:
There is a workaround solution already: https://twitter.com/ApacheAirflow/status/1337415983447814144

**Comment by kaxil** [comment]
Just posting the solution here too, install `importlib-metadata` and it should work fine:


```
pip install importlib-metadata~=2.0
```

**Comment by kaxil** [comment]
The commit for this has been merged to v1-10-test and if we create 1.10.15, this fix would be a part of it. If not, Airflow 2.0 will anyway contain the fix.

Commit: https://github.com/apache/airflow/commit/e75deee11ab8ed626979f1fe3927049a200ab676
